kong/router/atc.lua

```diff
@@ -423,6 +387,9 @@ do
 end
 
 
+local CACHE_PARAMS = {}
```

Comment:
Can we use https://github.com/Kong/kong/blob/master/kong/tools/request_aware_table.lua to detect race conditions? Should have no performance differences.

Reply:
`request_aware_table` needs to call `var.request_id`, it may have more cost.

Reply:
done. please review again.